Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -107,6 +120,11 @@ class TextureCacheBase
     u32 GetNumLevels() const { return texture->GetConfig().levels; }
     u32 GetNumLayers() const { return texture->GetConfig().layers; }
     AbstractTextureFormat GetFormat() const { return texture->GetConfig().format; }
+    u32 InMemoryFormat() const { return format & 0xff; }
```

Comment:
Switched it back to 0xf, the change to a different mask should probably happen in the hybrid PR